### ISSUE TITLE
heart-rate log missing data at end of day

### ISSUE THREAD

**Issue by tahnok** [tahnok/colmi_r02_client#22]
It seems like the last few entries in a day are not being parsed properly or returned.

A sample capture that I think should have data at 11:00 and 11:30 is

```
b'\x15\x00\x18\x05\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x002'
b'\x15\x01@\xaf!g\x00\x00\x00\x00\x00\x00\x00\x00\x00\x8d'
b'\x15\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x17'
b'\x15\x03\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x18'
b'\x15\x04\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x19'
b'\x15\x05\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x1a'
b'\x15\x06\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x1b'
b'\x15\x07\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x1c'
b'\x15\x08\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x1d'
b'\x15\t\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x1e'
b'\x15\n\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x1f'
b'\x15\x0b\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00 '
b'\x15\x0c\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00!'
b'\x15\r\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00M\x00\x00o'
b'\x15\x0e\x00\x00\x00\x00\x00\x00\x00\x00\x00G\x00\x00\x00j'
b'\x15\x0f\x00\x00W\x00\x00\x00\x00\x00Z\x00\x00\x00\x00\xd5'
b'\x15\x10\x00b\x00\x00\x00\x00\x00Y\x00\x00\x00\x00\x00\xe0'
b'\x15\x11G\x00\x00\x00\x00\x00j\x00\x00\x00\x00\x00D\x1b'
b'\x15\x12\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00L\x00s'
b'\x15\x13\x00\x00\x00\x00I\x00\x00\x00\x00\x00K\x00\x00\xbc'
b'\x15\x14\x00\x00\x00\x00\x00\x00\x00\x00\x00R\x00\x00\x00{'
b'\x15\x15\x00\x00T\x00\x00\x00\x00\x00o\x00\x00\x00\x00\xed'
b'\x15\x16\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00+'
b'\x15\x17F\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00r'
```

**Comment by tahnok** [comment]
More sample data from discord: https://discord.com/channels/1295372197036163145/1295372197749330007/1301328036591046716

**Comment by tahnok** [comment]
fixed in 01ba299b5f0b10646f67be6823966ca7aa31dd60
